app/concepts/reservations/use_cases/cancel_unpaid_reservations.rb

```diff
@@ -9,7 +9,11 @@ def initialize(repository: Reservations::Repository.new)
       def call(screening_id)
           reservations = @repository.unpaid(screening_id)
           reservations.map do |reservation|
+            client = Clients::Repository.new.find_by(reservation.client_id)
+            ReservationMailer.with(reservation: reservation, email: client.email).cancellation_email.deliver_now
```

Comment:
That's more of a suggestion that actual something you need to change at this point, but:
As you have already configured sidekiq, you can use `deliver_later` here - that way email will be sent as async job, which may reduce request times. Default queue on which mailers run is called... well, `mailers` so you will need to probably either change it to `default` or just configure sidekiq to process that queue as well. But it is something extra, you can skip that for now